iosxr_iso2vbox.py

```diff
@@ -493,6 +496,10 @@ def define_vbox_vm(vmname, base_dir, input_iso):
         run(['VBoxManage', 'modifyvm', vmname,
              '--nic' + str(i), 'nat', '--nictype' + str(i), 'virtio'])
 
+    # logger.debug('Enable packet capture on Mgmt NIC')
```

Comment:
I like this. Do you think it's worthy of another option (--trace) rather than comment it out?